test/index.js

```diff
@@ -166,4 +166,19 @@ describe('onBuild()', () => {
 
     expect(await pathExists(`${PUBLISH_DIR}/subdir/dummy.txt`)).toBeTruthy()
   })
+
+  test.each([
+    { FUNCTIONS_SRC: 'functions', resolvedFunctions: 'functions' },
+    { FUNCTIONS_SRC: undefined, resolvedFunctions: 'netlify-automatic-functions' },
+  ])('copy files to the functions directory', async ({ FUNCTIONS_SRC, resolvedFunctions }) => {
+    await useFixture('functions_copy_files')
+    await plugin.onBuild({
+      constants: {
+        FUNCTIONS_SRC,
+        PUBLISH_DIR: '.',
+      },
+    })
+
+    expect(await pathExists(`${resolvedFunctions}/function/index.js`)).toBeTruthy()
```

Comment:
huge nit but could possibly be helpful for any devs new to the project:

could be helpful to have the fixture `functions_copy_files` be 

`out_functions/next_random/next_random.js`
and 
`out_functions/next_random/nextJsPage.js`

just to mimic exactly what NoN generates

Reply:
Good point! Fixed.